api-specs/api_spec_PGS.yaml

```diff
@@ -721,8 +721,8 @@ paths:
               $ref: '#/components/schemas/CreditCardResumeRequest'
         required: true
       responses:
-        "302":
-          description: "FOUND, Redirect to url"
+        "200":
+          description: "OK, executed asynchronous call to vpos"
```

Comment:
Is this change correct in the context of this PR?

Reply:
Yes, it is correct

Reply:
Yes, you are right i confused the PRs.
I can change it to "OK", such that the 2 api-specs line up

Reply:
Yes please, let's keep PRs self-consistent 😃

Reply:
i merged https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/200 into develop,  and i merged develop into this branch